tests/python_client/testcases/test_collection.py

```diff
@@ -113,6 +113,23 @@ def test_collection_illegal_name(self, name):
         self.collection_wrap.init_collection(name, schema=default_schema, check_task=CheckTasks.err_res,
                                              check_items=error)
 
+    @pytest.mark.tags(CaseLabel.L2)
+    @pytest.mark.parametrize("name", ["_co11ection$","co11_$uper"])
```

Comment:
A blank between two values in the list is preferred: ["_co11ection$","co11_$uper"]

Reply:
> A blank between two values in the list is preferred: ["_co11ection$","co11_$uper"]

Added.